libraries/tools/kotlin-gradle-plugin/src/main/kotlin/org/jetbrains/kotlin/gradle/targets/js/nodejs/NodeJsSetupTask.kt

```diff
@@ -24,7 +26,7 @@ open class NodeJsSetupTask : DefaultTask() {
     init {
         @Suppress("LeakingThis")
         onlyIf {
-            settings.download && !env.nodeBinDir.isDirectory
+            settings.download && !env.nodeExecutable.isFile
```

Comment:
`nodeExecutable` now is only String, not File
Maybe we need to wrap `env.nodeExecutable` in constructor call of File?

Reply:
Sorry for the delay, done now.